Tests/test_Seq_objs.py

```diff
@@ -203,8 +220,31 @@ def test_str_count(self):
     def test_str_count_overlap_GG(self):
         """Check our count_overlap method using GG."""
         # Testing with self._examples
-        expected = [3, 3, 3, 3, 1, 1, 1, 1, 0, 0, 0, 0,  # Seq() Tests
-                    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]  # UnknownSeq() Tests
+        expected = [
+            3,
+            3,
+            3,
+            3,
+            1,
+            1,
+            1,
+            1,
+            0,
+            0,
+            0,
+            0,  # Seq() Tests
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+        ]  # UnknownSeq() Tests
```

Comment:
The comment placement is now unhelpful - perhaps:

```python
        expected = [
            # Seq() Tests:
            3,
            3,
            3,
            3,
            1,
            1,
            1,
            1,
            0,
            0,
            0,
            0, 
            # UnknownSeq() Tests:
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
        ]
```

Reply:
Yes, I didn't know what to do with it, I have removed.